db/models.py

```diff
@@ -1,4 +1,9 @@
+from django.contrib.auth.models import AbstractUser, User
```

Comment:
The import of `User` from `django.contrib.auth.models` is unnecessary since you are defining a custom `User` model inheriting from `AbstractUser`. Consider removing this import to avoid confusion.